dataland-frontend/src/components/resources/dataTable/modals/GdvYearlyDecimalTimeseriesModal.vue

```diff
@@ -0,0 +1,195 @@
+<script setup generic="KeyList extends string" lang="ts">
+import { computed, inject, type Ref } from "vue";
+import { type DynamicDialogInstance } from "primevue/dynamicdialogoptions";
+import {
+  type GdvYearlyDecimalTimeseriesDataConfiguration,
+  type MappedOptionalDecimal,
+  type YearlyTimeseriesData,
+} from "@/components/resources/dataTable/conversion/gdv/GdvYearlyDecimalTimeseriesData";
+
+import DataTable from "primevue/datatable";
+import Column from "primevue/column";
+import ColumnGroup from "primevue/columngroup";
+import Row from "primevue/row";
+import { formatNumberToReadableFormat } from "@/utils/Formatter";
+
+interface GdvYearlyDecimalTimeseriesDialogData<T extends string> {
+  label: string;
+  input: YearlyTimeseriesData<MappedOptionalDecimal<T>>;
+  options: GdvYearlyDecimalTimeseriesDataConfiguration<T>;
+}
+
+type GdvDecimalTimeseriesRow<T> = {
```

Comment:
Reminder for mayself: Check that vertical space is removed

Reply:
I removed the vertical space between the year and the text field on the upload page just now.

Reply:
ok